### DISCUSSION TITLE
Is is possible to initialize the TourService twice with different steps?

### DISCUSSION THREAD

**Discussion by julianogv** [hakimio/ngx-ui-tour#58] [General]
I have a initial tour starting when the user open the app for the first time which is working fine. Now I would like to show a second tour when the user access an specific route of my app.

What would be the recommended approach to implement the TourService.initialize with new steps is this scenario? 
Should I initialize it first when the app starts and again when an specific app route is accessed and start the service again?

**Comment by hakimio** [comment]
Yes, reinitializing the tour with different steps should work as expected. Let me know if you experience any issues doing that.
